### ISSUE TITLE
A note on reproducibility - changes in Uniprot IDs

### ISSUE THREAD

**Issue by rafwiewiora** [choderalab/ensembler#84]
A word of caution about using Uniprot IDs.

I have come back to my Ensembler set-up from a few months ago, to find that:
```
ensembler gather_targets --gather_from uniprot --query SETD8_HUMAN --uniprot_domain_regex SET
ensembler gather_templates --gather_from uniprot --query SETD8_HUMAN
```

no longer work. It's already not reproducible!

Uniprot appears to have changed the ID from SETD8_HUMAN to KMT5A_HUMAN. I've now learnt that I should be using the accession number (AC), not IDs - ACs never change (http://www.uniprot.org/help/difference_accession_entryname).

**Comment by jchodera** [comment]
Yikes!

Does ensembler offer a way to retrieve by accession number?

For reproducibility, we can still let you search by uniprot ID but it could generate an input file where these are translated into accession numbers.

**Comment by danielparton** [comment]
Yep, definitely use accession numbers if you need to use persistent IDs.
The `--query` argument is passed directly to the UniProt search API, which
allows you to search by accession number as follows:
`--query 'accession:Q9NQR1'`
http://ensembler.readthedocs.io/en/latest/examples.html#example-using-the-main-pipeline-functions

**Comment by rafwiewiora** [comment]
Thanks Danny, exactly what I needed!
